libs/sdk-core/src/models.rs

```diff
@@ -271,13 +287,15 @@ pub struct LnPaymentDetails {
     pub bolt11: String,
 }
 
+/// Details for a payment which closed a channel
 #[derive(PartialEq, Eq, Debug, Clone, Deserialize, Serialize)]
-pub struct ClosesChannelPaymentDetails {
+pub struct ClosedChannelPaymentDetails {
     pub short_channel_id: String,
     pub state: ChannelState,
     pub funding_txid: String,
 }
 
+/// LN channel managed by the LSP
```

Comment:
Not managed and doesn't have to be with the LSP peer. I would use just an "LN channel"

Reply:
Done